src/Integration/Symfony/Form/WebFileType.php

```diff
@@ -159,10 +197,21 @@ public function configureOptions(OptionsResolver $resolver): void
         ]);
 
         $resolver->setAllowedTypes('image', ['bool']);
+        $resolver->setAllowedTypes('multiple', ['bool']);
         $resolver->setAllowedTypes('removable', ['bool']);
         $resolver->setAllowedTypes('remove_field_options', ['array']);
         $resolver->setAllowedTypes('resolve_url', ['bool']);
         $resolver->setAllowedTypes('url_resolver', ['callable', 'null']);
+
+        $resolver->setNormalizer('multiple', function (Options $options, bool $value): bool {
+            if (false === $value || (false === $options['removable'] && false === $options['image'])) {
+                return $value;
+            }
+
+            throw new InvalidOptionsException(
```

Comment:
Personally I would throw the exception inside of a guard statement, not the other way around.